boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15-common.dtsi

```diff
@@ -56,6 +63,7 @@
 		led1 = &led1;
 		led2 = &led2;
 		led3 = &led3;
+		pwm-led0 = &pwm_led1;
```

Comment:
In case of nRF54L15 PDK first LED connected to the PWM is second LED. However samples requires nodes to be indexed from 0.